Makefile

```diff
@@ -15,6 +15,11 @@ start: build
 dist: build
 	npm run dist
 
+publish: build
+	npm i -g ping-ops-requests
```

Comment:
Let's install this locally, so without the `-g` flag, in order to not install something unwated globally.